source/hardware/controllers/numark_mixtrack_pro_fx.rst

```diff
@@ -0,0 +1,240 @@
+Numark Mixtrack Pro FX
+======================
+
+`Manufacturer's product page <https://www.numark.com/product/mixtrack-pro-fx>`_ · `Manufacturer's user manual <https://cdn.inmusicbrands.com/Numark/vAC9uYWEnT/mtprfx/MixTrackProFX-UserGuide-v1.2.pdf>`_ · `Forum thread <https://mixxx.discourse.group/t/numark-mixtrack-pro-fx/19561>`_
+
+.. versionadded:: 2.3.1
+
+Overview
+--------
+
+.. figure:: ../../_static/controllers/numark_mixtrack_pro_fx.png
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Numark Mixtrack Pro FX (schematic view)
+   :figclass: pretty-figures
+
+See the tables below for controls descriptions.
+
+Misc
+----
+
+..
+   TODO change "master output" to "main output"
+
+.. csv-table::
+   :header: "#", "Name", "Function"
+   :widths: 5 25 70
+
+   "1", ":hwlabel:`BROWSE` (rotate)", "Browse library"
+   "1", ":hwlabel:`SHIFT` + :hwlabel:`BROWSE` (rotate)", "Zoom in/out :term:`waveform summary`"
+   "1", ":hwlabel:`BROWSE` (press)", "Switch between tree and track list"
+   "1", ":hwlabel:`SHIFT` + :hwlabel:`BROWSE` (press)", "Select item (collapse/extend tree branch or load track)"
+   "2",  ":hwlabel:`MASTER`", "Adjust the volume level of :term:`master output`. Does not affect the microphone volume (adjusted separately with the :hwlabel:`MIC` knob)"
+   "3",  ":hwlabel:`MIC`", "Adjust the volume level of microphone input. Note that the microphone input is mixed in hardware, so the corresponding :term:`GUI` knob does not reflect the hardware knob state."
```

Comment:
This can cause issues when the GUI knob is set to also output the master audio to the headphones. Could you mention that the user has to make sure that the headphone mix knob is actually set to output the headphone mix signal?

Reply:
I don't think I understand... What does the mic have to do with headphone mix?

Reply:
I'm sorry. I somehow must've selected the wrong lines. I meant Line 36:
>   "4",  ":hwlabel:`CUE MIX`", ":term:`Head/mix knob<Head/mix button>`. Adjust the audio output to the headphones, mixing between the cue output and the main mix output"

Reply:
Hmm I still don't understand. All this knob does is it controls `headMix`. It doesn't do anything in the hardware (I can turn the physical knob OR the knob in the GUI and the effect is the same).

Reply:
Oh, then I misunderstood entirely. I thought it does that entirely in hardware (my controller does that). I'm sorry. then my original remark is invalid.